src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private final StringBuilder array = new StringBuilder();
```

Comment:
You need to have a fresh instance of the array for each run of getStatistic to pass the test of repeatedMethodCall, so you can remove final and initialize it in getStatistic method, but better option is to get rid of the field altogether, change your methods from void to returning value, and pass that value to next methods - create report, write to file